charts/aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -117,7 +117,7 @@ spec:
             - name: ADDRESS
               value: /csi/csi.sock
             - name: DRIVER_REG_SOCK_PATH
-              value: /var/lib/kubelet/plugins/ebs.csi.aws.com/csi.sock
+              value: {{ printf "%s/plugins/ebs.csi.aws.com/csi.sock" (trimSuffix "/" .Values.node.kubeletPath) }}
```

Comment:
Hi @makhov, thank you for your contribution! 
To address #1274, can you please include this change in [charts/aws-ebs-csi-driver/templates/node.yaml](charts/aws-ebs-csi-driver/templates/node.yaml) as well? Specifically, in [volumeMounts](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/b05ed2d4fc67e7986cbd026c448b349911908af4/charts/aws-ebs-csi-driver/templates/node.yaml#L87) and [volumes](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/430194761f903a21097ea01d0a9ca21b34d99ed6/deploy/kubernetes/base/node.yaml#L110).

Reply:
Volumes and mounts are correct. `DRIVER_REG_SOCK_PATH` should contain the path on the host, not on the container.

> `--kubelet-registration-path`: This is the path to the CSI driver socket on the host node that kubelet will use to issue CSI operations

https://github.com/kubernetes-csi/node-driver-registrar#required-arguments

Reply:
That fix is not to deal with the registry socket - because, as you pointed out, that path is relative to the host. The issue is that the kubelet attempts to mount volumes inside its own directory. Thus, with this fix the driver will be able to start up but will not mount volumes correctly because the kubelet will ask for volume mounts at (for example) `/var/lib/k0s/kubelet/volumes/...`, but the driver will have `/var/lib/kubelet` mounted instead of `/var/lib/k0s/kubelet`. Both changes are needed to properly support a modified kubeletPath.

Reply:
Ok, now I see the point: kubelet sends a request to the driver containing the real path on the host node, so we need to mount it into the pod with the proper path.

Thanks! I fixed the volumeMount, volume [was already fixed](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/69e98ed251c0b12cb0f3c2108680a3d5bed74948/charts/aws-ebs-csi-driver/templates/node.yaml#L157).